FWCore/AbstractServices/src/ResourceInformation.cc

```diff
@@ -3,4 +3,13 @@
 namespace edm {
   ResourceInformation::ResourceInformation() = default;
   ResourceInformation::~ResourceInformation() = default;
+
+  HardwareResourcesDescription ResourceInformation::hardwareResourcesDescription() const {
```

Comment:
This function should be moved to the concrete implementation, so that it gets loaded only via the plugin manager.